docs/adrs/0073-using-templates-for-sql-queries.md

```diff
@@ -0,0 +1,79 @@
+---
+title: 0073 Use Templates for SQL Queries
+description: |
+  Decision outcome: Establish a pattern for SQL query template artifacts
+---
+
+# Use Templates for SQL Queries
+
+**User Story:** *[MB-12577](https://dp3.atlassian.net/browse/MB-12577)* :lock:
+
+## Problem Statement
+The Go method which fetches a move's data change history contains a very large SQL query in which it sets up the data that is being tracked by the database triggers whenever data changes in tables. 
+The problem with such a large SQL statement in Go code is that it is essentially a string with no SQL linting or checking. This also means that developers lose the benefit of potential syntax highlighting when writing SQL queries.
+
+## Decision Drivers
+#### Do not ~~try to reinvent the wheel~~ circumvent Pop - [ADR 0001](docs/adrs/0001-go-orm.md)
+
+While evaluating potential options and opportunities, one key consideration is how the chosen approach will work with the existing ORM architecture [(Pop)](https://github.com/gobuffalo/pop).
+
+In order to achieve the best return value with the least effort, the chosen approach should not require excessive refactoring or removal of the Pop ORM pattern if at all possible.
+
+## Considered Alternatives
+
+> **bold denotes chosen**
+
+* *Do nothing*
+* *Utilize a SQL Template Engine*
+* *Build out our own SQL Templating architecture*
+* **A hybrid approach using an external library implemented to work with Pop**
+
+## Decision Outcome
+
+<!-- * Chosen Alternative:  -->
+* Chosen Alternative: *A hybrid approach using an external library implemented to work with Pop*
+* Positive Outcomes: 
+  * We will have a SQL templating pattern that is easy to implement without requiring major refactor, provides us with linting opportunities and better code visualization, and may be very useful in the future if we end up implementing additional complex queries.
+* Consequences: 
+  * If this new templating is not utilized elsewhere we will likely continue defaulting to writing SQL queries as strings in the Go code, which is our current pattern. 
+* Other considerations:
+  * The templating pattern must be clearly documented in order to be easily adopted.
```

Comment:
This doesn't need to be addressed here, but how would you like to approach this? We definitely need to socialize this more. I suggest [the #g-database channel in DP3 Slack](https://ustcdp3.slack.com/archives/CSGDM3NUW)🔒.

Reply:
I was thinking I would follow up with another PR adding a "how to" documentation article to docusaurus, do you think that makes the most sense?

Reply:
absolutely. But let's hold off on this since you've made the post in Slack about socializing it more. We can certainly tackle it in another sprint down the road. I think your ADR is documentation enough. But I also have transcom/mymove#9207 that tries to add the minimal amount of documentation around this. I think we're good enough for now.